packages/redux-toolbelt-observable/src/makeAsyncEpic.js

```diff
@@ -1,12 +1,13 @@
 import { from, throwError, of } from 'rxjs'
 import { ofType } from 'redux-observable'
-import { mergeMap, map, catchError } from 'rxjs/operators'
+import { mergeMap, switchMap, map, catchError } from 'rxjs/operators'
 
-export default function makeAsyncEpic(actionCreator, asyncFn) {
+export default function makeAsyncEpic(actionCreator, asyncFn, cancelPreviousRequests = false) {
```

Comment:
I'd change it that the last parameter is an object called `options`,
So that if they add more options to `makeAsyncEpic` then it'd all be in that object, and not as more function parameters.

So basically:

```javascript
export default function makeAsyncEpic(actionCreator, asyncFn, {cancelPreviousRequests = false} = {}) {
   ...
}
```

So if you pass nothing (`makeAsyncEpic(abc, xyz)`) the third parameter defaults to `{}` and `cancelPreviousRequests` will be `false`.